tests/install_upgrade_operators/console_cli_download/test_disconnected_virtctl.py

```diff
@@ -94,8 +94,8 @@ def test_download_and_execute_virtcli_binary_linux(self, downloaded_and_extracte
 @pytest.mark.arm64
 class TestDisconnectedVirtctlAllLinksInternal:
     @pytest.mark.polarion("CNV-6915")
-    def test_all_links_internal(self, all_virtctl_urls, non_internal_fqdns):
+    def test_all_links_internal(self, all_virtctl_urls_scope_function, non_internal_fqdns):
```

Comment:
to have duplicate fixture just for this test isnot needed, you can use `all_virtctl_urls_scope_class` here as well